libursa/DatabaseUpgrader.cpp

```diff
@@ -38,11 +38,27 @@ void save_json(std::string_view path, const json &dbjson) {
     fs::rename(tmp_db_name, path);
 }
 
-void upgrade_v1_0_0(json *dbjson) { (*dbjson)["version"] = "1.3.2"; }
+void upgrade_v1_0_0(json *dbjson) {
+    json &db = *dbjson;
+    db["version"] = "1.3.2";
+
+    if (db.find("config") != db.end()) {
+        db["config"] = std::map<std::string, std::string>();
+    }
+
+    if (db["config"].find("max_mem_size") != db["config"].end()) {
+        db["config"].erase("max_mem_size");
+    }
+
+    if (db["iterators"] == nullptr) {
+        db["iterators"] = std::map<std::string, std::string>();
```

Comment:
```suggestion
        db["iterators"] = std::unordered_map<std::string, std::string>();
```


This requires to `#include <unordered_map>`!